code/WorkInProgress/SomepotatosStuff.dm

```diff
@@ -117,17 +117,17 @@
 	set name = "GPS"
 	set category = "Special Verbs"
 	set desc = "Find your way around with ease!"
-	if(PROC_ON_COOLDOWN(10 SECONDS))
-		boutput(src, "Verb on cooldown for [time_to_text(PROC_ON_COOLDOWN(0))].")
+	if(ON_COOLDOWN(src, /mob/living/carbon/verb/GPS, 10 SECONDS))
+		boutput(src, "Verb on cooldown for [time_to_text(ON_COOLDOWN(src, /mob/living/carbon/verb/GPS, 0))].")
 		return
 	if(hasvar(src,"wear_id"))
 		DoGPS(src:wear_id)
 /mob/living/silicon/verb/GPS()
 	set name = "GPS"
 	set category = "Special Verbs"
 	set desc = "Find your way around with ease!"
-	if(PROC_ON_COOLDOWN(10 SECONDS))
-		boutput(src, "Verb on cooldown for [time_to_text(PROC_ON_COOLDOWN(0))].")
+	if(ON_COOLDOWN(src, /mob/living/silicon/verb/GPS, 10 SECONDS)) // using ..... is very wacked
```

Comment:
:(
But it's so cool! It works automatically, like magic!

Reply:
its really cursed but i'll look at supporting it in spacemandmm because no matter what work arounds i tried the spacemandmm parser just doesn't like `....` or (the correct) `.....`

as a note it currently uses `....` which if you try the following example code, isnt correct:
```
world.log << "[....]"
----
code.dm:3:error: ....:  compile failed (possible infinite cross-reference loop)

VS

world.log << "[.....]"
----
/a/New
```

Reply:
It absolutely is wacky and breaks with indentation (?) levels. If you use `..... ` here it doesn't compile and it actually needs to be `....`. I don't think anyone really knows how it works.

Reply:
`code\WorkInProgress\SomepotatosStuff.dm:120:error: .....: not a prototype`